src/kani-compiler/src/codegen_cprover_gotoc/codegen/assert.rs

```diff
@@ -2,38 +2,60 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
 //! This file contains the code that acts as a wrapper to create the new assert and related statements
-
 use crate::codegen_cprover_gotoc::GotocCtx;
 use cbmc::goto_program::{Expr, Location, Stmt};
-use std::str::FromStr;
 
 /// The Property Class enum stores all viable options for classifying asserts, cover assume and other related statements
-#[derive(Copy, Debug, Clone)]
+#[derive(Debug, Clone)]
+#[allow(dead_code)]
 pub enum PropertyClass {
     ExpectFail,
+    Unimplemented,
+    ExactDiv,
+    SanityCheck,
     UnsupportedStructs,
+    Unreachable,
+    Cover,
+    PointerOffset,
+    AssertFalse,
+    Assume,
     DefaultAssertion,
+    CustomProperty(String),
```

Comment:
Should this be InternedString

Reply:
sure, will change it to use that.

Reply:
Forgot to update this but we don't use `InternedString` on the codegen side of things so I just left it as `String` for now.